webapp/src/ts/modules/tasks/tasks.component.ts

```diff
@@ -116,14 +130,13 @@ export class TasksComponent implements OnInit, OnDestroy {
     window.location.reload();
   }
 
-  private hydrateEmissions(taskDocs) {
+  private async hydrateEmissions(taskDocs) {
```

Comment:
No `await`, no need of `async`
```suggestion
  private hydrateEmissions(taskDocs) {
```